package.json

```diff
@@ -18,23 +18,24 @@
     "test": "ember try:each"
   },
   "dependencies": {
-    "active-model-adapter": "2.1.1",
-    "ember-bootstrap-controls": "git+https://github.com/wildland/ember-bootstrap-controls#v0.12.0",
-    "ember-cli-babel": "^5.1.7",
+    "active-model-adapter": "2.2.0",
+    "ember-cli-babel": "^6.6.0",
     "ember-cli-htmlbars": "^1.1.1",
     "ember-router-generator": "^1.0.0",
-    "torii": "^0.8.0"
+    "torii": "^0.10.1"
+  },
+  "peerDependencies": {
+    "ember-bootstrap-controls": "git+https://github.com/wildland/ember-bootstrap-controls#v1.0.0-beta.0"
```

Comment:
Recommend also having ember auto-install this library if it is a peer dependency on install, which can be done here: https://github.com/wildland/ember-authenticate-me/blob/master/blueprints/ember-authenticate-me/index.js